x_xy/io/xml/abstract.py

```diff
@@ -111,49 +111,33 @@ def to_xml(element: T, pos_min: jax.Array, pos_max: jax.Array):
         element.set("pos_max", _to_str(pos_max))
 
 
-def _from_xml_vispy(attr: ATTR):
-    """Find all keys starting with `vispy_`, and return subdict without that prefix.
-    Also convert all arrays back to list[float], because of `struct.field(False)`.
-    Otherwise jitted functions with `sys` input will error on second execution, since
-    it can't compare the two vispy_color arrays.
-    """
```

Comment:
After having read some comments below, then read this: If we do require that conversion to list[float], please then also include this comment from here down there as some inline comment (can be much shorter). Because this was an annoying bug to find back then.

Reply:
updated `_from_xml_geom_attr_processing` to convert jax.Arrays to tuples for color and edge_color